addons/mysterious_egg/static/src/builder/plugins/media_website_plugin.js

```diff
@@ -0,0 +1,54 @@
+import { Plugin } from "@html_editor/plugin";
+import weUtils from "@web_editor/js/common/utils";
```

Comment:
don t use web_editor.
Maybe move this utils in this file. We only need it for website builder (maybe ?)